tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -808,18 +819,19 @@ public void ViewAreProhibitedTest([Values(false, true)] bool async)
 
             void AssertResult(Exception ex)
             {
-                var isLinux = RequirePlatform.GetCurrentOperatingSystem() == SupportedOperatingSystem.Linux;
+                var currentOperatingSystem = RequirePlatform.GetCurrentOperatingSystem();
+                var canThrowPlatformNotSupported = currentOperatingSystem != SupportedOperatingSystem.Windows;
```

Comment:
Rename variable to `shouldThrowPlatformNotSupportedException`

Reply:
done